lib/kapost_deploy/plugins/notify_honeybadger_after_promote.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "kapost_deploy/heroku/app_releases"
+
+module KapostDeploy
+  module Plugins
+    # After-promotion plugin to notify honeybadger after a promotion is complete.
+    # Honeybadger.io can be configured to clear all honeybadgers upon this deploy notification.
+    class NotifyHoneyBadgerAfterPromote
+      def initialize(config,
+                     ahead_releases: KapostDeploy::Heroku::AppReleases.new(config.app, token: config.heroku_api_token))
+        self.config = config
+        self.git_config = config.options.fetch(:git_config, nil)
+        self.ahead_releases = ahead_releases
+      end
+
+      def before
+      end
+
+      def after
+        notify_honeybadger
+      end
+
+      private
+
+      attr_accessor :config, :git_config, :ahead_releases
+
+      def notify_honeybadger
+        system("bundle exec honeybadger deploy -e #{env} -s #{commit_sha} -r #{repository_url}")
+      end
+
+      def env
+        config.to.split("-").last
+      end
+
+      def commit_sha
+        ENV["CIRCLE_BUILD_NUM"] || pipeline_sha
```

Comment:
We should change `CIRCLE_BUILD_NUM` to `CIRCLE_SHA1`.  Forgot that in arugula, the `CIRCLE_BUILD_NUM` is not actually the correct value.  We want to be sending the sha1 hash.